openpype/hosts/blender/api/plugin.py

```diff
@@ -36,6 +36,12 @@ def prepare_scene_name(
     if namespace:
         name = f"{name}_{namespace}"
     name = f"{name}_{subset}"
+
+    # Blender name for a collection or object cannot be longer than 63
+    # characters. If the name is longer, it will raise an error.
+    if len(name) > 63:
+        raise ValueError(f"Asset name '{name}' is too long.")
```

Comment:
```suggestion
        raise ValueError(f"Scene name '{name}' would be too long.")
```

Reply:
I still think we should somehow solve it if possible, @antirotor any ideas?

I was thinking to do something like add 3 dots at the start of name?
```python
if len(name) > 63:
    name = f"...{name[:-60]}"
```

Reply:
To reduce the string size without losing uniqueness, wouldn't a hash function be a solution ?

Reply:
Sadly not, because it's Blender to handle the names of the libraries when you load one with `bpy.data.libraries.load`, and as far as I know, you cannot rename one!